packages/component-tree-to-app/src/keyCallbackMap.ts

```diff
@@ -182,4 +182,10 @@ export const keyCallbackMap: {
       actions: [{ type: "do_nothing" }],
     },
   },
+  Tabs: {},
+  Textarea:{},
+  Search:{},
+  Select:{},
+  Typography:{},
+  Title:{},
```

Comment:
We don't need `packages/component-tree-to-app` package anymore. Hence, no need to update it.